style.css

```
@@ -0,0 +1,202 @@
+  color: #000;
+  margin-top: -200px;
+  font-size: 16px;
+}
+
+input:focus {
+  border-color: blue;
+}
+
+footer {
+  text-align: center;
+  text-decoration: none;
+  margin-top: 21rem;
+  color: #505050;
+  font-size: 10px;
+}
+
+footer a {
+  text-align: center;
+  text-decoration: none;
+  color: #505050;
+}
+
+h1 {
+  color: #4c4c4c;
+  font-weight: 400;
+  line-height: 1.1;
+  font-size: 1.4rem;
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.topSign {
+  grid-column: 3;
+  grid-row-start: 1;
+  border-radius: 4px;
+  border: 1px solid #4c4c4c;
+  background: #0077c5;
+  color: white;
+  margin-right: 2rem;
+  font-size: 1.125em;
+  font-weight: normal;
+  padding: 0 20px 0 20px;
+}
+
+.alText {
+  grid-column: 2;
+  grid-row: 1;
+  padding-right: 1rem;
+  margin-top: 2px;
+  margin-bottom: 1rem;
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: normal;
+  color: #505050;
+}
+
+.fPass {
+  grid-column: 2;
+  grid-row: 1;
+  padding-right: 1rem;
+  padding-top: 1rem;
+}
+
+.links {
+  display: grid;
+  grid-template-columns: repeat(3, 1);
+  grid-template-rows: auto;
+  grid-gap: 2rem;
+  align-content: center;
+  margin-right: 32rem;
+  margin-left: 39.5%;
+  margin-bottom: 1.5rem;
+  max-width: 380px;
+  min-width: 380px;
+}
+
+.intuitLogo {
+  margin-left: 1rem;
+  place-self: center;
+  grid-column: 2;
+  grid-row: 1;
+}
+
+.mint {
+  grid-column: 1;
+  grid-row: 2;
+  place-self: end;
+}
+
+.quickbooks {
+  grid-column: 2;
+  grid-row: 2;
+}
+
+.turbotax {
+  grid-column: 3;
+  grid-row: 2;
+}
+
+.oneAcc {
+  text-align: center;
+  font-size: 14px;
+  line-height: 0.8em;
+}
+
+.EPP {
+  text-align: left;
+  font-size: 14px;
+  padding-bottom: 4px;
+  font-weight: normal;
+  color: #404040;
+  line-height: 25px;
+}
+
+.EPP2 {
+  font-size: 10px;
+  color: #444;
+  text-align: left;
+}
+
+.signIn {
+  float: center;
+  text-align: center;
+  grid-gap: 1rem;
+  border-bottom: none;
+  margin-bottom: 10rem;
+  margin-left: 36%;
+  min-width: 480px;
+  max-width: 480px;
+  max-height: 10rem;
+  background: #fff;
+  overflow: visible;
+}
+
+.signIn > div {
+  border: 1px solid #c7c7c7;
+  border-bottom: none;
+  background: #fff;
+  padding: 2rem;
+}
+
+.signIn > div:last-child {
+  border: 1px solid #c7c7c7;
+  border-top: none;
+  background-color: #eceef1;
+  color: grey;
+  font-style: italic;
+  position: relative;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  text-align: center;
+}
+
+.reCAPTCHA {
+  font-size: 0.8rem;
+  font-style: italic;
+  text-align: center;
+  background-clip: content-box;
+}
+
+.createAccount {
+  background-color: #0077c5;
+  color: #fff;
+  border-radius: 6px;
+  font-size: 1.27em;
+  min-width: 100px;
+  padding: 12px 125px;
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.italicText {
+  font-style: italic;
+  color: grey;
+  text-align: center;
+}
```

add a class that is going to be in the section tag, you have a class called body but should be something diferent like container since body is a name of a tag.  this class should handle the center of all content with the margin, and  the flex display in column with the align-items will center the 2 elements that you have.
```suggestion
}

.container {
   margin: 0 auto;
   display: flex;
   flex-direction: column;
   align-items: center;
}
```